RiotSwiftUI/Modules/UserSessions/Common/View/UserSessionCardViewData.swift

```diff
@@ -86,7 +86,7 @@ struct UserSessionCardViewData {
         case .unverified:
             return isCurrentSessionDisplayMode ? VectorL10n.userSessionUnverifiedAdditionalInfo : VectorL10n.userOtherSessionUnverifiedAdditionalInfo + " %@"
         case .permanentlyUnverified:
-            return VectorL10n.userOtherSessionPermanentlyUnverifiedAdditionalInfo
+            return isCurrentSessionDisplayMode ? VectorL10n.userOtherSessionPermanentlyUnverifiedAdditionalInfo : VectorL10n.userOtherSessionPermanentlyUnverifiedAdditionalInfo + " %@"
```

Comment:
I can see this is following the example above and we don't support RTL in EI so it isn't an issue, but really the placeholder should be part of the string itself so that translators could place it wherever makes most sense for their language.

Reply:
Yeah, it makes sense!
This time we didn't to this way to avoid to double the localized strings I guess (we use the same copy with the placeholder and without it in some cases).